include/deal.II/base/vectorization.h

```diff
@@ -3400,6 +3428,15 @@ class VectorizedArray<double, 2>
     data = _mm_loadu_pd(ptr);
   }
 
+  DEAL_II_ALWAYS_INLINE
+  void
+  load(const float *ptr)
+  {
+    DEAL_II_OPENMP_SIMD_PRAGMA
+    for (unsigned int i = 0; i < 2; ++i)
+      data[i] = ptr[i];
+  }
```

Comment:
@kronbichler Any suggestions for SSE2? I think it would be fine this way.

Reply:
There is also some SSE functions for the conversion, but it is not straight-forward because one needs to access 64 bytes from a 128 byte array, which needs some special operations, so let's leave it like this, or even without the pragma.